src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -1,7 +1,11 @@
 package seedu.address.logic.commands;
 
+//import java.util.List;
```

Comment:
I dont think you should comment out, just delete it. Importing it is also a quick action